.github/actions/upload-release-archive/action.yml

```diff
@@ -0,0 +1,37 @@
+name: Upload release archive
+description: Bundles an archive artifacts and uploads to a GitHub release. The artifacts must already have been built into the folder "install".
+inputs:
+  os:
+    description: The OS name being built, used in the archive name
+    required: true
+runs:
+  using: composite
+  steps:
+    - name: archive
+      id: archive
+      shell: bash
+      run: |
+        OSNAME=$(echo ${{ inputs.os }} | awk '{print tolower($0)}')
```

Comment:
It looks like `| sed 's/-latest//')` changed to `| awk '{print tolower($0)}` here?

Reply:
Sorry thought `runner.os` produced the same, but it was `linux` instead of `ubuntu`, reverted and kicked off build to verify

https://github.com/anuraaga/wabt/actions/runs/5238769469